File: GCD.py
def gcd(a, b):
    smaller = a if a < b else b
    result = 1
    for i in range(2, smaller + 1):
        if a % i == 0 and b % i == 0:
            result = i
    return result

x = int(input("Enter first integer: "))
y = int(input("Enter second integer: "))
print(f"GCD of {x} and {y} is {gcd(x, y)}")

File: GCD.java
import java.util.Scanner;

public class GCD {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter first integer: ");
        int a = input.nextInt();
        System.out.print("Enter second integer: ");
        int b = input.nextInt();

        int gcd = 1;
        int smaller = (a < b) ? a : b;

        for (int i = 2; i <= smaller; i++) {
            if (a % i == 0 && b % i == 0) {
                gcd = i;
            }
        }

        System.out.println("GCD of " + a + " and " + b + " is " + gcd);
    }
}